### PR TITLE
Formatting fixes in contextlib page

### PR DESCRIPTION
**Pull request by slateny**
Removed and added some `>>>`/`...` depending on whether it had sample outputs. removed unneeded whitespace, added whitespace to make example 4-space indentation

### CODE REVIEW

File: Doc/library/contextlib.rst [hunk 1]
```diff
@@ -454,18 +453,17 @@ Functions and classes provided:
...
-              print('Starting')
-              return self
-
-          async def __aexit__(self, *exc):
-              print('Finishing')
-              return False
+      >>> from asyncio import run
+      >>> from contextlib import AsyncContextDecorator
 
+      >>> class mycontext(AsyncContextDecorator):
```
Comment (JelleZijlstra) [inline Doc/library/contextlib.rst]: I'm a little sad that this makes the code harder to copy-paste, but agree that this improves consistency. I guess the idea in the existing code was to have no `>>>` for the code defining the sample classes, only for the code using them. I am going to look at the rest of the docs to see whether that's an established convention.

Comment (slateny) [reply Doc/library/contextlib.rst]: If that was the idea, then I think the code block can just be split into two with something like 'managed_resource can then be used like the following:', and have the second chunk contain the `>>>`

Comment (slateny) [reply Doc/library/contextlib.rst]: i.e.: 
image

Comment (JelleZijlstra) [reply Doc/library/contextlib.rst]: I think that would be better, thanks!

**Review by JelleZijlstra — APPROVED** [review]

**Pull request merged**